e2e/encoding/index.ts

```diff
@@ -21,7 +21,7 @@ const FunctionalView: FunctionalComponent = () =>
 
 const router = createRouter({
   // TODO: allow hash based history
-  history: createWebHistory('/' + __dirname),
+  history: createWebHistory('/encoding'),
```

Comment:
can't `__dirname` be replaced with BASE_URL? https://vitejs.dev/guide/env-and-mode.html#env-variables

Reply:
![image](https://user-images.githubusercontent.com/17949342/118064942-9f2b9300-b3ce-11eb-931c-9e61898a5778.png)

Reply:
https://github.com/vitejs/vite/pull/2939#pullrequestreview-632945950